apps/edxapp/templates/services/nginx/bc_nginx.yml.j2

```diff
@@ -0,0 +1,45 @@
+{%- from "apps/edxapp/templates/services/cms/macros/image.yml.j2" import image_name -%}
+{%- set service="nginx" -%}
```

Comment:
I don't see the interest to have this definition here.

Reply:
ditto